oshi-core/src/main/java/oshi/software/os/mac/MacNetworkParams.java

```diff
@@ -30,6 +30,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.sun.jna.Native;
```

Comment:
(Optional) The `// NOSONAR` comment suppresses warnings from SonarQube for using `com.sun` imports (although JNA should be an exception).  If you can move those comments up from 2 lines below (it needs to be on the first import) it will be helpful.